content/en/security/threats/workload_security_rules/custom_rules.md

```diff
@@ -19,6 +19,14 @@ This topic explains how to create custom Datadog Agent and detection rules for [
 
 In addition to the out of the box (OOTB) [default Agent and detection rules][7], you can write custom Agent and detection rules. Custom rules help to detect events Datadog is not detecting with its OOTB rules.
 
+## RBAC for custom rule management
+
+To prevent users with the [Datadog Standard out-of-the-box role][11] from creating/updating, disabling, and deploying a changed rule via remote configuration, do the following:
```

Comment:
```suggestion
To prevent users with the [Datadog Standard out-of-the-box role][11] from creating, updating, disabling, and deploying a changed rule using remote configuration:
```